amazon-kinesis-client/src/main/java/software/amazon/kinesis/retrieval/polling/SimpleRecordsFetcherFactory.java

```diff
@@ -32,7 +32,7 @@
     private int maxPendingProcessRecordsInput = 3;
     private int maxByteSize = 8 * 1024 * 1024;
     private int maxRecordsCount = 30000;
-    private long idleMillisBetweenCalls = 1500L;
+    private long idleMillisBetweenCalls = 200L;
```

Comment:
Can you check if the canary TPS is getting impacted after this change?

Reply:
NVM      
`Note: There is no throttling enforced on this Activity to ensure no get calls Archiver makes on behalf of the
 customer are counted against customer's public throttling limits.`